src/Services/FacilityService.php

```diff
@@ -353,12 +353,10 @@ public function getOne($uuid)
             "limit" => 1
         ));
 
-        if ($sqlResult) {
```

Comment:
just checking that sqlResult is always populated with something?

Reply:
Yes, this was the reason PHP7.4 and PHP8.0 were breaking. 
Got it from here. 
https://stackoverflow.com/questions/59674903/trying-to-access-array-offset-on-value-of-type-bool-in-php-7-4

Reply:
in this case, would revert your changes here and do the following for the if:
```
if (!empty($sqlResult)) {
```

Reply:
@vishnuyar , still need to address this issue

Reply:
Done